nix/manual-overrides.nix

```diff
@@ -56,6 +61,8 @@ hself: hsuper: {
   # Explicitly enable haddock because cabal2nix disables it for packages with
   # internal libraries
   cql-io = hlib.doHaddock (hlib.dontCheck hsuper.cql-io);
+  connection = null;
```

Comment:
have you tried removing this? I don't think, it's needed, if anything, set it to `crypton-connection`, perhaps?